### ISSUE TITLE
a11y tag clean-up

### ISSUE THREAD

**Issue by InnaAtanasova** [SAP/fundamental-styles#1580]
We need to go over the components having the a11y tags and verify if all a11y requirements are met. Only in this case the tag should be kept, o.w. should be removed.

List of components having the a11y tag:
- [ ] Action Bar
- [ ] Avatar
- [ ] Bar
- [ ] Breadcrumb
- [ ] Busy Indicator
- [ ] Calendar
- [ ] Card
- [ ] Carousel
- [ ] Forms
- [ ] Dialog
- [ ] Feed input
- [ ] File Uploader
- [ ] Generic Tile
- [ ] Info Label
- [ ] Link
- [ ] List
- [ ] Menu
- [ ] Message Box
- [ ] Message Page
- [ ] Message Strip
- [ ] Message Toast
- [ ] Notification
- [ ] Pagination
- [ ] Popover
- [ ] Product Switch
- [ ] Shellbar
- [ ] Side Nav
- [ ] Tabs
- [ ] Time
- [ ] Toolbar
- [ ] Tree
- [ ] Combobox Input
- [ ] Contextual Menu
- [ ] Multi Input
- [ ] Time Picker
- [ ] Page
